### ISSUE TITLE
Some languages reported as unknown even though they're suggested

### ISSUE THREAD

**Issue by TeMPOraL** [chubin/cheat.sh#32]
Observe:

  temporalgalactica~@> curl cht.sh/lisp
  Unknown topic.
  Do you mean one of these topics may be?
  
      * lisp/ 100
      * elisp/ 89
      * alias 67

Changing the URL to cht.sh/alias correctly gives a list of examples, while lisp and elisp are always reported as unknown. My guess is, something somewhere is storing those tokens with forward slash in the name, whereas the HTTP handling code strips it out from request.

**Comment by chubin** [comment]
Yes, it's true, we are experiencing too high load now, and that it the root of the problem.
We are working on it and as soon as it will be fixed, everything should work fine.

Please try, whether `curl cht.sh/lisp/:list` works for you.

**Comment by TeMPOraL** [comment]
Yes, `:list` works. It returns `:learn`. `:learn` returns a Lisp guide.

Thanks, I'll keep an eye on it and play some more once the load goes down :).

**Comment by chubin** [comment]
Yes, I will write about it in this issue, as soon as the problem is fixed/mitigated.
